runtime/testy/studentAPI.py

```diff
@@ -64,9 +64,13 @@ class Gamepad(StudentAPI):
 
   def __init__(self, toManager, fromManager):
     super().__init__(toManager, fromManager)
+    self._get_gamepad()
+
+  def _get_gamepad(self):
+    self.all_gamepads = self._getSMValue("gamepads")
```

Comment:
The current code on master doesn't have one, and I don't think we need one? What would the 0 be for.